components/resources/brave_components_resources.grd

```diff
@@ -61,6 +61,7 @@
         <part file="../brave_perf_predictor/resources/brave_perf_predictor_resources.grdp" />
       </if>
 
+      <part file="playlist_resources.grdp" />
```

Comment:
we have some inconsistencies here, but seems like this should be ../playlist/resources/playlist_resources.grdp like perf predictor above?